### PR TITLE
Address UX feedback tweaks.

### PR DESCRIPTION
**Pull request by prodonjs**
- Shortens sidebar links and thins divider to 1px.
- Adds secondary text to platform links
- Fixes hover / active state for dashboard card links.
- Address a11y issue for tabbing on iframe-links.
- Fixes an invalid selector in main-page unit tests.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3356)

### CODE REVIEW

**Comment by prodonjs** [comment]
/cc @avdaredevil 
/cc @ajayalfred 
/area front-end
/area centraldashboard

This addresses the majority of the tweaks we talked about yesterday afternoon save for the font-weight: 500 issue.

I didn't deploy it in cluster, but I recorded this [screencast](https://screencast.googleplex.com/cast/NTk0NTQ2NTA0NjU2NDg2NHwxOTg1YjQyNS1hMg) showing the correct hover and active behaviors along with the adding secondary text lines for the platform cards.

File: components/centraldashboard/public/components/dashboard-view.css [hunk 1]
```diff
@@ -40,32 +55,39 @@ paper-card {
         font-weight: 500;
         padding: 0 1em;
         flex: 1;
-    };
+    }
+    ;
```
Comment (avdaredevil) [inline components/centraldashboard/public/components/dashboard-view.css]: This is really weird formatting...

File: components/centraldashboard/public/components/dashboard-view.css [hunk 2]
```diff
@@ -15,15 +17,28 @@
...
+    grid-column: 3
+}
+
 paper-card {
     border-radius: 5px;
     width: 100%;
     overflow: hidden;
-    min-width: 23em; /* 320px */
+    min-width: 23em;
+    /* 320px */
```
Comment (avdaredevil) [inline components/centraldashboard/public/components/dashboard-view.css]: This is really weird formatting

File: components/centraldashboard/public/components/main-page_test.js [hunk 3]
```diff
@@ -124,8 +124,8 @@ describe('Main Page', () => {
         });
 
     it('Appends query string when building links', () => {
-        const sidebarLinkSelector = '#MainDrawer iron-selector a.iframe-link';
-        const headerLinkSelector = 'app-header paper-tabs a';
+        const sidebarLinkSelector = '#MainDrawer iron-selector iframe-link';
+        const headerLinkSelector = '#ViewTabs paper-tabs a';
```
Comment (avdaredevil) [inline components/centraldashboard/public/components/main-page_test.js]: Nice! 💯

**Comment by prodonjs** [comment]
Latest commit adds the font-weight support which looks like this.

Screenshot from 2019-05-24 12-25-16

I'll try to clean up the automatic CSS formatting that my IDE is using. I don't think we have a standard template that's enforced anywhere so mine is probably using whatever the Google default is.

**Comment by avdaredevil** [comment]
@prodonjs looks great, however regarding your point about style guide.
Mix-ins in CSS are not part of standard CSS, therefore your processor is
Mis-formatting.

**Comment by ajayalfred** [comment]
Hover states look fantastic! Thanks Jason. Glad to see font-weight: 500 can be loaded. Card headers look much better.

Quick note on the left nav font-styles

**Active state CSS (nav item currently selected):**
font-family: GoogleSans;
font-weight: 500;
font-size: 15px;
color: #FFFFFF;
letter-spacing: 0.3px;
line-height: 24px;

**Inactive state CSS (all other nav items):**
opacity: 0.5;
font-family: GoogleSans;
font-weight: 500;
font-size: 15px;
color: #FFFFFF;
letter-spacing: 0.3px;
line-height: 24px;

**Comment by ajayalfred** [comment]
Change 'Overview' to 'Home' in nav

**Comment by prodonjs** [comment]
@ajayalfred Thanks for the feedback. I addressed those comments and pushed the image to https://kubeflow-1.endpoints.prodonjs-kubeflow-dev.cloud.goog/. PTAL when you get the chance.

@avdaredevil I figured it was related to the Polymer-specific mix-ins because I just use the standard beautify formatting plugin. It'd be great if there was a way to have a formatter that respects the mix-in syntax, but for now I just disabled formatting on save within this project.

**Comment by ajayalfred** [comment]
Looks fantastic!! Really digging the changes with Google sans weight: 500 and the nav font changes!!

One final cosmetic improvement. I've noticed that the contrast between the gray canvas and the light-blue container for the GCP card isn't sufficient. They kind of blend into each other. See below.

image

I think it might be best to use the standard card header treatment but with the GCP logo and blue color. See below.
image

Hope this is an easy fix!

**Comment by avdaredevil** [comment]
This should just be a one-line change. I think @prodonjs already tackled some of my AI from the UX meeting, so this can be part of it.

**Comment by prodonjs** [comment]
Latest change removes the background and also makes the line-height for the GCP links consistent with the quick shortcuts. 

**Before**
Screenshot from 2019-05-28 13-24-47


**After**
rffzXGot2i9

**Comment by ajayalfred** [comment]
Perfect! thanks!

**Comment by prodonjs** [comment]
@avdaredevil Latest commit adds the aria-role we discussed this morning. Let me know if there's anything else.

**Pull request merged**